styles/views/attitude-bar-view.css

```diff
@@ -0,0 +1,94 @@
+.attitudeBarView.block{
+  margin-top: 10%;
+  position: absolute;
+}
+
+.attitudeBarView.block.component{
+  margin-top: 550px;
+}
+
+.attitudeBarView.block.component.yaw{
+  margin-left: 10%;
+}
+.attitudeBarView.block.component.pitch{
+  margin-left: 45%;
+}
+.attitudeBarView.block.component.roll{
+  margin-left: 80%;
+}
+.attitudeBarView.input{
+  //margin-top: 30%;
+  width: 40px;
+}
+.attitudeBarView.send{
+  //margin-top: ;
+  left: 10%;
+}
+
+.attitudeBarView.text{
+  position: absolute;
+}
+
+.attitudeBarView.block.front{
+  background-color: rgba(32,32,32,1);
+  color: rgba(222,222,222,1);
+  text-align: center;
+  height: 200px;
+  width: 100px;
+}
+
+.attitudeBarView.block.back{
+  height: 400px;
+  width: 100px;
+  background-color: rgba(244,233,177,1);
+}
+
+.attitudeBarView.yaw.block{
+  //margin-top: 10%;
+  margin-left: 10%;
+}
+
+.attitudeBarView.pitch.block{
+  //margin-top: 10%;
+  margin-left: 45%;
+  margin-right: 45%;
+  //display: none;
+}
+
+.attitudeBarView.roll.block{
+  //margin-top: 10%;
```

Comment:
This is invalid css (not how you make comments in css, not should you keep commented code in pull requests.)